kitchen-tests/Gemfile

```diff
@@ -1,13 +1,9 @@
 source "https://rubygems.org"
 
-gem "chef-config", "< 13"
 gem "rake" # required to build some native extensions
 gem "berkshelf"
```

Comment:
you could try pulling berks from master here

Reply:
Do we need to do that or *should* we just to test that

Reply:
its a _should_ not a _need_, although technically i think we should nuke the Gemfile.lock and pull from git master on everything we control (unless there's some issue requiring pinning)